share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml

```diff
@@ -149,15 +143,11 @@ spack:
     +tempus +tpetra +trilinoscouplings +zoltan +zoltan2 +superlu-dist gotype=long_long
   - turbine
   - umap
-  - umpire
   - upcxx
-  - veloc
-  - vtk-m
   - wannier90
-  - zfp
 
   # CUDA
-  - adios2 +cuda cuda_arch=80
+  - ecp-data-vis-sdk +adios2 +ascent +cinema +cuda +darshan +faodel +hdf5 +paraview +pnetcdf ~rocm ~sensei +sz +unifyfs +veloc +visit +vtkm +zfp cuda_arch=80
```

Comment:
Paraview needs to be disabled here, there are known issues when building with cuda.

Reply:
done!